build/scripts/do-release-candidate.sh

```diff
@@ -1,20 +1,17 @@
 #!/bin/sh -ex
 
-if [ -z "${CIRCLE_PULL_REQUEST}" ] && [ -n "${CIRCLE_TAG}" ] && [ "${CIRCLE_PROJECT_USERNAME}" = "weaveworks" ] ; then
-  export RELEASE_DESCRIPTION="${CIRCLE_TAG}"
-  RELEASE_NOTES_FILE="docs/release_notes/${CIRCLE_TAG/-rc.*}.md"
+if [ -z "${GITHUB_REF_NAME}" ] || [ "${GITHUB_REF_TYPE}" != "tag" ] ; then
+  echo "Expected a tag push event, skipping release workflow"
+  exit 1
+fi
 
-  if [[ ! -f "${RELEASE_NOTES_FILE}" ]]; then
-    echo "Release notes ${RELEASE_NOTES_FILE} not found. Exiting..."
-    exit 1
-  fi
+RELEASE_NOTES_FILE="docs/release_notes/${GITHUB_REF_NAME/-rc.*}.md"
 
-  goreleaser release --rm-dist --timeout 60m --skip-validate --config=./.goreleaser.yml --release-notes="${RELEASE_NOTES_FILE}"
+if [ ! -f "${RELEASE_NOTES_FILE}" ]; then
+    echo "Release notes file ${RELEASE_NOTES_FILE} does not exist. Exiting..."
+    exit 1
+fi
 
-  sleep 90 # GitHub API resolves the time to the nearest minute, so in order to control the sorting oder we need this
+export RELEASE_DESCRIPTION="${GITHUB_REF_NAME}"
```

Comment:
where is `RELEASE_DESCRIPTION` used?

Reply:
It's used in the `.goreleaser.yml` template.